nox.py

```diff
@@ -245,7 +245,7 @@ def session_lint(session):
     """Lints each sample."""
     session.install('flake8', 'flake8-import-order')
     session.run(
-        'flake8', '--builtin=gettext', '--max-complexity=10',
+        'flake8', '--builtin=gettext', '--max-complexity=15',
```

Comment:
![tumblr_inline_nzb6a3hkjg1rp5ay0_500](https://cloud.githubusercontent.com/assets/250995/17258435/b6e96e06-557b-11e6-9846-5d6810235517.gif)

(it's actually the large switch statement `if __main__` that it doesn't like)